input/postgres/establish_connection.go

```diff
@@ -66,20 +66,32 @@ func connectToDb(ctx context.Context, config config.ServerConfig, logger *util.L
 				dbPasswordOverride = dbToken
 			}
 		} else if config.SystemType == "google_cloudsql" {
-			if config.GcpProjectID == "" || config.GcpRegion == "" || config.GcpCloudSQLInstanceID == "" {
-				return nil, errors.New("To use IAM auth with Google Cloud SQL, you must specify project ID, region, and instance ID")
+			// if config.GcpProjectID == "" || config.GcpRegion == "" || config.GcpCloudSQLInstanceID == "" {
+			// 	return nil, errors.New("To use IAM auth with Google Cloud SQL, you must specify project ID, region, and instance ID")
```

Comment:
Because we can't totally distinguish CloudSQL from AlloyDB without a `systemType`, I'm not sure we can do much here. Without this check, we assume that if a user has added the `db_use_iam_auth` with either, they mean it and have followed the configuration instructions.

Reply:
Interestingly, i just saw this conversation when helping CME group on another issue. Maybe there is value in adding "alloydb" as a `systemType` for clarity in the UI as well?

[https://pganalyze.zendesk.com/agent/tickets/6569](https://pganalyze.zendesk.com/agent/tickets/6569)